### ISSUE TITLE
The detail conduction of experiments. 

### ISSUE THREAD

**Issue by Thinkre** [ASolitaryMan/Foal-Net#2]
I tried to reproduce your reported results and finally got 79.23% of UA and 77.69% of WA. 
The settings are same with in the paper you reported. 8 gpus are used for distributed training. 
For each fold, four sessions are used to train and one session is used to validated and tested. And the model is random initiated and trained from a scratch. I saved the best model according to UA and output predicted labels. 
Finally, I cocat all predicted labels together to evaluate the metric of UA and WA. 
I don't where is wrong.

**Comment by ASolitaryMan** [comment]
Hello, due to differences in machine configurations, experimental environments, and limitations of dataset sizes, I cannot guarantee the exact replication of results. I suggest you use 4 GPUs for training, with a batch size of 16 for each GPU (4*16=64), which is the setting I used in my paper. Additionally, I realized that I did not upload the requirements.yml file. I will upload the requirements.yml file for my experimental environment later.

**Comment by ASolitaryMan** [comment]
Moreover, you should change the value of max_length to 6,  ( "parser.add_argument('--max_length', type=int, default=14, help='max length of audio')) and change the value of epoch is 40.

**Comment by Thinkre** [comment]
Thanks a lot. I will try again. 
I have configured the experimental environments already. 
It seems that the argment of max_length is not used actually. It was fixed, see the line 142 to 148 in the file of dataloader.py 
Did you concat all the prediction together to evaluate or just do average of metrics over 5 sessions?  Although, the result was very close.

**Comment by ASolitaryMan** [comment]
I do average of metrics over 5 sessions.
